static-site/app/blog/[id]/page.tsx

```diff
@@ -81,9 +99,13 @@ export async function generateMetadata({
   return metadata;
 }
 
+/**
+ * A React Server Component for rendering a single blog post
+ */
 export default async function BlogPost({
   params,
 }: {
+  /** the string identifing the post we're generating metadata for */
```

Comment:
Suggestion: remove description

```suggestion
```

This description doesn't seem accurate (`BlogPostParams` is not a string) nor necessary (the only thing in `params` is `params.id` which is documented [elsewhere](https://github.com/nextstrain/nextstrain.org/blob/495b6948e22c399b3e0974e57fee6ac718203022/static-site/app/blog/%5Bid%5D/page.tsx#L18-L21)).